### ISSUE TITLE
[IATI] Og file XML > spreadsheet output

### ISSUE THREAD

**Issue by stevieflow** [OpenDataServices/cove#1208]
This is mentioned in #959 and also via https://github.com/OpenDataServices/flatten-tool/issues/151 - but I wanted to log a specific issue in terms of the input of an Org XML file into CoVE does result in schema and ruleset tests - and then offers a conversion to spreasheet

As documented in #959 - this generates a blank spreadsheet

It would be good to fix this

**Comment by Bjwebb** [comment]
@stevieflow Here's a dev deploy — http://1208-iati-cove-orgxml-spreadsheet.dev.cove.opendataservices.coop/ — could you check that's behaving as expected?

**Comment by robredpath** [comment]
@stevieflow When you get a sec, could you check this so that we can close this issue if it's working as expected?

**Comment by Bjwebb** [comment]
I think we got the feedback we needed from @stevieflow on the [pull request](https://github.com/OpenDataServices/cove/pull/1219).

**Comment by stevieflow** [comment]
Yeah, thanks @Bjwebb - I think we have that in the PR @robredpath (thanks for checking though)
